public/app/features/logs/components/logParser.ts

```diff
@@ -1,9 +1,12 @@
+import { partition } from 'lodash';
```

Comment:
Do we have enough coverage of these changes to feel confident? Asking mainly because the last P1 incident was produced by this code.

Reply:
hmmm.. i think for the "current" dataframes (not the "dataplane" ones) the tests are good enough...
we updated the tests a lot, just not directly in this PR (i tried to update the tests first, to be sure this PR does not break anything.. that's why there are no test changes in this PR):
- https://github.com/grafana/grafana/pull/71922 (this tests for the incident you mentioned)
- https://github.com/grafana/grafana/pull/71941
- https://github.com/grafana/grafana/pull/71840

regarding the "dataplane" mode, i will add some tests for that case 👍 ... (though this is the new mode, no datasource emits these frames currently, so nothing could break)

WDYT?

Reply:
(i added tests for the "dataplane-mode" scenario too)